accounting/filters/OsgScheddCpuFilter.py

```diff
@@ -154,7 +155,7 @@ def schedd_collector_host(self, schedd):
                 # Cache the CollectorHost in the map
                 if "CollectorHost" in ads[0]:
                     schedd_collector_hosts = set()
-                    for schedd_collector_host in ads[0]["CollectorHost"].split(","):
+                    for schedd_collector_host in re.split(', |,| ', ads[0]["CollectorHost"]):
```

Comment:
Your regex captures `, ` or `,` or ` `. It's possible that there could be more than one space separating values, which means empty captures will happen. I suggest using `[, ]+` for the regex, which will split on any number of consecutive spaces and/or commas separating items:
```python
>>> collector_string = "one,two, three four  five,  six"
>>> re.split(r', |,| ', collector_string)
['one', 'two', 'three', 'four', '', 'five', '', 'six']
>>> re.split(r'[, ]+', collector_string)
['one', 'two', 'three', 'four', 'five', 'six']
```

Using "raw" strings (`r"Hello World"`) is also generally recommended when writing regexes in Python (see [the intro section in the docs](https://docs.python.org/3/library/re.html)), though since there are no slashes in this regex, it's not a big deal..

```suggestion
                    for schedd_collector_host in re.split(r'[, ]+', ads[0]["CollectorHost"]):
```